salt/grains.j2

```diff
@@ -81,11 +81,7 @@ iscsi_name: "{{  iscsi.name if "iscsi" in env }}"
 
 iscsi_ip: "{{  iscsi.private_ip if "iscsi" in env }}"
 
-qdevice_enabled: {{ jsonify(qdevice.enabled) if "qdevice" in env else jsonify(false) }}

```

Comment:
if we remove this grains we should remove them from the `main` example file otherwise we endup with broken documentation API ( grains in our case).

see https://github.com/SUSE/pacemaker-deploy/blob/master/deployment.libvirt.yaml.example#L31